app/scripts/controllers/organization/AddHolController.js

```diff
@@ -117,6 +117,35 @@
                     location.path('/holidays');
                 });
             };
+            scope.submit=function(){
+			  var fromtxt=angular.element('#fromDate').val();
+			  var toDatetxt=angular.element('#toDate').val();
+			  var repaymentsRescheduleTotxt=angular.element('#repaymentsRescheduledTo').val();
+			  if(new Date(fromtxt)<new Date()||new Date(toDatetxt)<new Date()||new Date(repaymentsRescheduleTotxt)<new Date()){
+				alert("Invalid date(s) given");
```

Comment:
Make the standard errorStatus and errorArray work instead of pushing a popup alert.